test/Configuration.jl

```diff
@@ -38,4 +40,40 @@ end
     ["--compile=min", "--project=test", "--startup-file=no", "--history-file=no"]
 end
 
+@testset "authentication" begin
```

Comment:
Just to make sure that the tests are really testing authentication, and not a bad server/tests, could you change this test line to:

```julia
@testset "Authentication: $(auth ? "on" : "off")" for auth in [true, false]
```

and then enable/disable authentication accordingly, and use two different ports.

Reply:
Alternatively, you could leave authentication on for all tests, and instead add a second test where the authentication secret is provided in the URL, and check that this gives you access.